docs/gno-infrastructure/setting-up-a-local-chain.md

```diff
@@ -10,8 +10,59 @@ In this tutorial, you will learn how to start a local Gno node (and chain!).
 Additionally, you will see the different options you can use to make your Gno instance unique.
 
 ## Prerequisites
+- **Git**
+- **`make` (for running Makefiles)**
+- **Go 1.21+**
+- **Go Environment Setup**:
+  - Make sure `$GOPATH` is well-defined, and `$GOPATH/bin` is added to your `$PATH` variable.
 
-- [`gnoland` installed](local-setup.md#3-installing-other-gno-tools).
+## Installation
+
+To install the `gnoland` binary/tool, clone the Gno monorepo:
```

Comment:
It's a binary

Reply:
[80ab3b5](https://github.com/gnolang/gno/pull/2212/commits/80ab3b5aa09eb4fe1b99cca511a437c955f9ba0b)